integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.6.41 Version bump reason: [K32W0] SDK 2.6.10 update
+0.6.42 Version bump reason: [Java] download and install kotlin compiler
```

Comment:
With the PR as is, we do not need to bump the version as this would be a brand new stand-alone image.

However: we likely want to ensure chip-build-vscode also has the kotlin compiler so that we can build kotlin stuff from the generic "contains all" image. So please add this to that image and then the version change 100% makes sense since an existing image will be updated.

This would also allow cloudbuild integration to build test kotlin stuff.

Reply:
Added